Patch by Robert Scheck <robert@fedoraproject.org> for libvmime >= 0.7.1, which adds a missing include file
in order to build successfully with glibc 2.11.90 (2010-02-08) and newer. There was a glibc header cleanup
which no longer gets the sigset_t definition by accident, so the signal.h needs to be included explicitly.

--- libvmime-0.7.1/vmime/platforms/posix/posixChildProcess.hpp		2005-05-25 21:14:54.000000000 +0200
+++ libvmime-0.7.1/vmime/platforms/posix/posixChildProcess.hpp.sigset	2010-02-14 02:14:11.000000000 +0100
@@ -22,6 +22,7 @@
 
 
 #include "vmime/utility/childProcess.hpp"
+#include <signal.h>
 
 
 namespace vmime {
